### DISCUSSION TITLE
Multiple fields for sorting in `useSortBy` hook

### DISCUSSION THREAD

**Discussion by jsdmc** [algolia/instantsearch#5936] [Q&A]
Hi guys.
I'm using `react-instantsearch-hooks-web` with typesense. Is there a way to specify multiple sortings fields for search results?
Thanks

**Comment by Haroenv** [comment]
You'll have to ask that question on https://github.com/typesense/typesense-instantsearch-adapter

**Comment by jsdmc** [comment]
Asked here. So indeed few sort fields can be specified delimited by comma
https://github.com/typesense/typesense-instantsearch-adapter/issues/189
